### PR TITLE
Scanning secrets in environment variables blog

### PR DESCRIPTION
**Pull request by raulcabello**
Fix https://github.com/kubewarden/kubewarden.io/issues/135

### CODE REVIEW

File: content/blog/2022/10/env-var-secrets.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+authors:
+- Raul Cabello Martin
+date: 2022-10-17
+---
+
+We are thrilled to announce you can now scan your environment variables for secrets with the new
+[env-variable-secrets-scanner-policy](https://github.com/kubewarden/env-variable-secrets-scanner-policy)! This policy rejects a `Pod`
+or [workload resources](https://kubernetes.io/docs/concepts/workloads/) such as `Deployments`, `ReplicaSets`, `DaemonSets`
+, `ReplicationControllers`, `Jobs`, `CronJobs` etc. if a secret is found in the environment variable
+within a container, init container, or ephemeral container.
```
Comment (flavio) [inline content/blog/2022/10/env-var-secrets.md:12]: Please add a paragraph explaining how can find secrets that are leaked inside of base64 encoded env variables

File: content/blog/2022/10/env-var-secrets.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  - apiGroups: [""]
+    apiVersions: ["v1"]
+    resources: ["pods", "deployments", "replicasets", "daemonsets", "replicationcontrollers", "jobs", "cronjobs"]
+    operations:
+    - CREATE
+    - UPDATE
+EOF
+```
+
+Verify we are not allowed to create a Pod with an RSA private key
```
Comment (flavio) [inline content/blog/2022/10/env-var-secrets.md:53]:
```suggestion
Verify we are not allowed to create a Pod with a RSA private key
```

**Review by flavio — COMMENTED** [review]
Looks great to me, I left two minor comments

File: content/blog/2022/10/env-var-secrets.md (hunk 2 above)
Comment (viccuad) [reply content/blog/2022/10/env-var-secrets.md:53]: I would go with `an`, we are pronouncing the R as the letter (not as in radio), and R is vowel sound.

File: content/blog/2022/10/env-var-secrets.md (hunk 1 above)
Comment (raulcabello) [reply content/blog/2022/10/env-var-secrets.md]: added in https://github.com/kubewarden/kubewarden.io/pull/138/commits/f2ff7c000fd9c19a0ee8e5b5e60079ed760fcbba

**Review by jvanz — APPROVED** [review]

File: content/blog/2022/10/env-var-secrets.md (hunk 1 above)
Comment (flavio) [reply content/blog/2022/10/env-var-secrets.md]: thanks!

**Review by viccuad — APPROVED** [review]
agree, let's ship it! maybe needs a bump in the date

File: content/blog/2022/10/env-var-secrets.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Let's create a `ClusterAdmissionPolicy` that will scan all pods for secrets in their environment variables:
+
+```
+kubectl apply -f - <<EOF
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: env-variable-secrets
+spec:
+  module: ghcr.io/kubewarden/policies/env-variable-secrets-scanner-policy:v0.1.0
```
Comment (viccuad) [inline content/blog/2022/10/env-var-secrets.md:42]:
```suggestion
  module: ghcr.io/kubewarden/policies/env-variable-secrets-scanner:v0.1.1
```

Comment (viccuad) [reply content/blog/2022/10/env-var-secrets.md:42]: new location of the policy: https://github.com/kubewarden/env-variable-secrets-scanner-policy/pkgs/container/policies%2Fenv-variable-secrets-scanner

**Review by viccuad — CHANGES_REQUESTED** [review]

**Review by viccuad — APPROVED** [review]
Changed the policy url and version to the one without `-policy`.

**Pull request merged**